integrations/mysql.ini

```diff
@@ -3,7 +3,7 @@ RUN_MODE = prod
 
 [database]
 DB_TYPE  = mysql
-HOST     = 127.0.0.1:3306
+HOST     = mysql:3306
```

Comment:
This is not general, just for drone. We will not be able to run tests on local env (without modifying host). Same for pgsql.

Reply:
We will focus on ci integration for sure

Reply:
but could we run the test locally?

Reply:
Only if the value can be provided via some environment variable. But beside that you can also run drone exec locally.